_maps/readme.md

```diff
@@ -4,29 +4,23 @@
 
 The list of items that are modular requiring adding/modifying to /tg/ maps:
 
-- Positronic Alert Consoles (inside Robotics area)
-- Command Vendor (inside Meeting area)
-- LustWish Vendor (inside Recreational/Dormitory area)
 - Barbershop Room (including the vendors, chairs, mirrors, and more)
-- NanoTrasen Consultant's Office
-- Blueshield Office
 - Cryopods (inside Recreational/Dormitory area)
-- Stasis Beds to Stasis Pods
-- Armory/Gun Change
-- Prison Change
-- Security Medical Room
-- Arrival Tram and Console (inside Arrival area)
+- Security:
+- - Add Ammo Workbench
+- - Add Gun Vendor
+- Armory:
+- - Replace Disablers With Smartgun
+- - Replace Shotguns With Shotgun Spawners
+- - Replace Energy Guns With MCRS
+- - Replace Laser Guns With CMGs
+- Security Medics Room:
+- - Security Medics Spawner
+- - Security Medics Locker
+- Arrival Docks:
+- - Arrival Shuttle Docking Port Resize
+- - Arrival Shuttle Console
 - Central Command Ferry Hangar (inside Arrival area)
-- Update roundstart_template variable on mining/public/labour stationary docks: mining/x > mining/skyrat, mining_common/x > mining_common/skyrat, labour/x > labour/skyrat , mining/large > mining/large/skyrat
-- Replace the security outposts with their departmental guards
```

Comment:
I got a feeling this is going to cause issues with spawning the guards, and their roundstart equipment, if it's not properly carried over.

Reply:
They should be using sec guard spawns, if not, that needs changed

Reply:
My main concern would likely be the equipment lockers, in that case, then. Because I agree that they could use the same spawnpoints as the sec guard spawns, if possible.

Reply:
I don't even know what they get, but the lockers should also be updated in code if they aren't already, but, the stuff I am removing is stuff I haven't even been resetting for like... well, I've never included this stuff. And it's never been brought up as an issue

Reply:
Well someone should update the sec lockers to have this, but this is asking too much of priority